task/testmap.py

```diff
@@ -182,7 +182,8 @@
 # only put auxiliary images here; triggers for primary OS images are computed from testmap
 IMAGE_REFRESH_TRIGGERS = {
     "fedora-testing": [
-        "fedora-testing@cockpit-project/cockpit"
+        "fedora-testing@cockpit-project/cockpit",
```

Comment:
Is this necessary? I though that it would be computed. Or we don't consider `__manual` when computing contexts to be triggered?

Reply:
Correct, we ignore the `_manual` ones, because they are, well, *manual* :-)